FindTown/FindTownUI/Sources/FindTownUI/PopUp/AlertSuccessCancelPopUpViewController.swift

```diff
@@ -92,7 +92,9 @@ public class AlertSuccessCancelPopUpViewController: UIViewController {
         self.successButtonText = successButtonText
         self.cancelButtonText = cancelButtonText
         self.successButton.addAction(for: .touchUpInside) { _ in
-            successButtonAction?()
+            self.dismiss(animated: true) {
+                successButtonAction?()
+            }
```

Comment:
```suggestion
            self.dismiss(animated: false) {
                successButtonAction?()
            }
```

저는 팝업이 내려갈 때 부자연스러워서.. 로그아웃, 회원탈퇴 popup dismiss할 때 차라리 애니메이션을 빼는게 더 나은 것 같더라구요.. 😀 제 개인적인 생각입니당.. 😅

Reply:
네~ 애니메이션 빼는게 훨씬 자연스럽네요!!